gradle/libs.versions.toml

```diff
@@ -1,7 +1,7 @@
 [versions]
 kotlin = "1.8.20"
 kotlinx-coroutines = "1.7.1"
-kotlinx-serialization = "1.5.1"
+kotlinx-serialization = "1.5.0"
```

Comment:
This was causing problems when building a downstream project.

Reply:
Hmm this may use the newer 1.8.21 stdlib but shouldn't be bumping the gradle plugin or anything that would cause problems with using 1.8.20. Otherwise we would see the same problems here.